packages/decentraland-ecs/src/ecs/Engine.ts

```diff
@@ -363,12 +381,14 @@ export class Engine {
         system.removeEntity(entity)
       }
     }
+
+    return this
   }
 
   private componentAddedHandler(event: ComponentAdded) {
     const { entity, componentName } = event
 
-    if (!entity.isAddedToEngine()) return
+    if (!entity.isAddedToEngine()) return this
```

Comment:
```suggestion
    if (!entity.isAddedToEngine()) return
```
Private and unnecesary